### ISSUE TITLE
express.hbs compile issues with "noImplicitAny": true and using IOC module

### ISSUE THREAD

**Issue by 0v3rst33r** [lukeautry/tsoa#553]
## Sorting

* **I'm submitting a ...**
  - [X] bug report
  - [ ] feature request
  - [ ] support request

* I confirm that I
  - [X] used the [search](https://github.com/lukeautry/tsoa/search?type=Issues) to make sure that a similar issue hasn't already been submit

## Expected Behavior
When enabling the following setting in `tsconfig.json`, I expect no compiler errors from the `routes.ts` file.

## Current Behavior
Compile errors on the following code lines in `routes.ts`:
```ts
            if (typeof controller['setStatus'] === 'function') {
```
Error as follows:
```
TS7053: Element implicitly has an 'any' type because expression of type '"setStatus"' can't be used to index type 'XXXController'.
```

```ts
            if (typeof (<any>controller).setStatus === 'function') {
```

## Possible Solution

Change the line in `express.hbs` to:
```ts
if (typeof (<any>controller).setStatus === 'function') {
```
in order to prevent the compile error with `"noImplicitAny": true`

## Steps to Reproduce

1. Make use of the `IOC` module
2. Enable "noImplicitAny": true and compile

## Context (Environment)
Version of the library: 
Version of NodeJS: 
* Confirm you were using yarn not npm: [X]

## Detailed Description

Change the `if` check in `express.hbs` as indicated above:
```ts
if (typeof (<any>controller).setStatus === 'function') {
```

**Comment by 0v3rst33r** [comment]
I am creating a PR within the next few with the suggested change above, unless otherwise advised regarding a better solution/fix.

**Comment by dgreene1** [comment]
@0v3rst33r This was just released in v2.5.11 (https://github.com/lukeautry/tsoa/releases). Thank you

**Comment by 0v3rst33r** [comment]
@dgreene1 great, thanks! Appreciated (project `package.json` file already updated with `v2.5.11` and all happy again).
